nft/nfts.yaml

```diff
@@ -649,6 +649,105 @@ paths:
                     type: boolean
                     description: 'Whether the given wallet owns any token in the given NFT collection.'
       operationId: isHolderOfCollection
+  '/{apiKey}/getNFTSales':
+    get:
+      summary: getNFTSales
+      description: Gets NFT sales that have happened through on-chain marketplaces
+      tags: ['NFT API Endpoints']
+      parameters:
+        - $ref: '#/components/schemas/tokenId'
+        - $ref: nfts_body.yaml#/startBlock
```

Comment:
nfts_body.yaml no longer exists, you can reference things in the components/schemas section below!